### ISSUE TITLE
v5.1 doesn't generate tags

### ISSUE THREAD

**Issue by chrisjohnson00** [mathieudutour/github-tag-action#42]
Dependabot suggested I upgrade from `v5` to `v5.1` and following the upgrade I get...

```
2020-11-25T07:30:43.9712609Z ##[group]Run mathieudutour/github-tag-action@v5.1
2020-11-25T07:30:43.9713244Z with:
2020-11-25T07:30:43.9714368Z   github_token: ***
2020-11-25T07:30:43.9714710Z   release_branches: main
2020-11-25T07:30:43.9715050Z   default_bump: patch
2020-11-25T07:30:43.9715367Z   tag_prefix: v
2020-11-25T07:30:43.9715694Z   create_annotated_tag: false
2020-11-25T07:30:43.9716017Z   dry_run: false
2020-11-25T07:30:43.9716311Z ##[endgroup]
2020-11-25T07:30:44.2027246Z ##[group]Run actions/create-release@v1
2020-11-25T07:30:44.2027669Z with:
2020-11-25T07:30:44.2028034Z   release_name: Release 
2020-11-25T07:30:44.2028397Z   draft: false
2020-11-25T07:30:44.2028758Z   prerelease: false
2020-11-25T07:30:44.2029085Z env:
2020-11-25T07:30:44.2029841Z   GITHUB_TOKEN: ***
2020-11-25T07:30:44.2030175Z ##[endgroup]
2020-11-25T07:30:44.2796025Z ##[error]Input required and not supplied: tag_name
```
when I downgrade to `v5` again, it works perfectly.
There is no new tag created via `v5.1`

**Comment by boxcee** [comment]
@chrisjohnson00 What was the branch you pushed to?

**Comment by chrisjohnson00** [comment]
Main

**Comment by boxcee** [comment]
@chrisjohnson00 Sorry, cannot reproduce this yet.

Can you also share the workflow? The part where you are using the two actions?

**Comment by chrisjohnson00** [comment]
Sure, here you go

```
name: Create Release

on:
  push:
    branches: [ 'main' ]

jobs:
  create_release:
    runs-on: ubuntu-18.04
    name: Create Release
    steps:
      - uses: actions/checkout@v2
      - name: Bump version and push tag
        id: tag_version
        uses: mathieudutour/github-tag-action@v5
        with:
          github_token: ${{ secrets.REPO_ONLY }}
          release_branches: main
          default_bump: patch
      - name: Create a GitHub release
        uses: actions/create-release@v1
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        with:
          tag_name: ${{ steps.tag_version.outputs.new_tag }}
          release_name: ${{ steps.tag_version.outputs.new_tag }}
          body: ${{ steps.tag_version.outputs.changelog }}
```

The `REPO_ONLY` token is used because I need workflows to be triggered from the creation of the tag.

**Comment by chrisjohnson00** [comment]
Here's an example where the action fails:
https://github.com/chrisjohnson00/solaredge-prometheus-exporter/actions/runs/388022672

**Comment by boxcee** [comment]
Thanks.

**Comment by rnoda** [comment]
I'm in the same situation. Just upgraded to v5.1 to make use of the new 'custom_release_rules' input. However, no tag is being generated now, therefore the workflow is falling on a subsequent step that requires the new tag name.

**Comment by robinjhuang** [comment]
Same thing is happening to me. Works on v5

**Comment by AleksIvanovSinglet** [comment]
According to mathieu's changelog, the only thing different from v5 is that in the docs he references v5.1.

**Comment by chrisjohnson00** [comment]
> According to mathieu's changelog, the only thing different from v5 is that in the docs he references v5.1.
> 
> I can confirm also that v5 works.

Yes, but you'll also notice that `v5.1` was "released" yesterday, yet this Issue is 9 days old.  This means the `v5.1` release was deleted and re-created.

**Comment by mathieudutour** [comment]
no it wasn't, I just forgot to create release but the tag existed.

**Comment by chrisjohnson00** [comment]
This isn't fixed, as can be seen by issues #46 and #47 
This shouldn't have been closed, can you please re-open it @mathieudutour ?

**Comment by AleksIvanovSinglet** [comment]
> This isn't fixed, as can be seen by issues #46 and #47
> This shouldn't have been closed, can you please re-open it @mathieudutour ?

And also, Dependabot as expected, suggested an update to v5.1, which would break any workflow that merges that suggested PR.

**Comment by chrisjohnson00** [comment]
> This isn't fixed, as can be seen by issues #46 and #47
> This shouldn't have been closed, can you please re-open it @mathieudutour ?

@mathieudutour - was this fixed?
